src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Compile.xaml

```diff
@@ -38,6 +38,12 @@
       DisplayName="Custom Tool Namespace"
       Description="The namespace into which the output of the custom tool is placed." />
 
+    <StringProperty Name="FullPath" Visible="false">
```

Comment:
It represents the fully qualified path to the item, as used in the other items like Content...